From 166fc3c3b2db5fd6799d4e54d300c962b04fa3a3 Mon Sep 17 00:00:00 2001
From: Bruno Martins <bgcngm@gmail.com>
Date: Tue, 24 Jul 2018 11:20:31 +0100
Subject: [PATCH 22/26] config/common: Remove BluetoothExt from
 PRODUCT_PACKAGES

 * This package seems to be deprecated and it's no longer synced.

Change-Id: I903d7000593a68149ea9cd24d545ccc6fa399de4
---
 config/common.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/config/common.mk b/config/common.mk
index 8ffdb999..81015818 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -120,7 +120,6 @@ PRODUCT_PACKAGES += \
 
 # Required Lineage packages
 PRODUCT_PACKAGES += \
-    BluetoothExt \
     LineageParts \
     Development \
     Profiles
-- 
2.17.1

